build-gcc.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -e
+
+GCC_VERSION=10.4.0
+GCC_SHA256=ab1974017834430de27fd803ade4389602a7d6ca1362496c57bef384b2a4cb07
+PREFIX="${PREFIX:-/usr}"
```

Comment:
AFAICS all Dockerfiles set this to `/opt/gcc-${GCC_VERSION}`. Would it make sense to make that value default and not require passing it explicitly?

Reply:
I've set `PREFIX=/opt/gcc-${GCC_VERSION}` in `build-gcc.sh`.